configure.ac

```diff
@@ -372,6 +378,10 @@ dnl the manpages only if docbook2x-man is available.
 AC_PATH_PROG(DOCBOOK2X_MAN, docbook2x-man)
 AM_CONDITIONAL(HAVE_DOCBOOK2X_MAN, [test -n "${DOCBOOK2X_MAN}"])
 
+if test "${enable_wasm}" = yes ; then
```

Comment:
Are you sure this should not be like
`if test "${enable_wasm}" = "yes" ; then`
?

Reply:
It the same.